### ISSUE TITLE
Menu items with keyboard shortcut not list in shortcut options

### ISSUE THREAD

**Issue by jwrober** [longturn/freeciv21#1730]
**Is your feature request related to a problem? Please describe.**
We have come menu entries that show a keyboard short that are not listed in the shortcut options dialog box:

- Game -> Save Game == Ctrl+S
- View -> Native Tiles == Ctrl+Shift+N
- Select -> Select single unit == Ctrl+Z
- Select -> All on tile == V
- Select -> Same type on tile == Shift+V
- Select -> Same type on continent == Shift+C
- Select -> Same type everywhere == Shift+X
- Select -> Advanced unit selection == Ctrl+E
- Unit -> Unload all from transporter == Shift+U
- Unit -> Convert == Ctrl+O
- Unit -> Disband == Ctrl+D
- Combat -> Build fortess/buoy == Shift+F
- Combat -> Build Airbase == Shift+E
- Work -> Connect with Road == Ctrl+R
- Work -> Connect with Railroad == Ctrl+L
- Work -> Connect with Irrigation == Ctrl+I
- Work -> Help build wonder == B (duplicate)
- Work -> Establish Trade Route == R (duplicate)
- Multiplayer -> Delayed Goto == Z
- Multiplayer -> Delayed orders execute == Ctrl+Z
- Multiplayer -> Clear orders == Shift+Ctrl+C
- Multiplayer -> Add/Remove City == Ctrl+T
- Multiplayer -> Automatic Caravan == Ctrl+J
- Multiplayer -> Set/Unset Rally point == Ctrl+S
- Multiplayer -> Quick Airlift == Ctrl+Y 

We also have some menu items with no shortcut available, but makes sense to add one:

- Game -> Save Map to Image
- View -> City Buy Costs- 

**Describe the solution you'd like**
All keyboard shortcuts should be tailored to a players needs/wants.

**Describe alternatives you've considered**

**Additional context**
Discovered while working on #1726

**Comment by lmoureaux** [comment]
It would also be good to have some logic in the default shortcuts. For instance, we could reserve Ctrl for options that don't affect the game state.

**Comment by jwrober** [comment]
Good idea. Use Shift as a modifier for a base command. We do that in some but not all cases now.
